tests/components/mqtt/test_light_template.py

```diff
@@ -358,13 +663,31 @@
                     "command_topic": "test_light_rgb/set",
                     "command_on_template": "on,{{ transition }}",
                     "command_off_template": "off,{{ transition|d }}",
+                    "qos": 1,
                 }
             },
         )
 
     state = hass.states.get("light.test")
     assert state.state == STATE_OFF
 
+    assert state.attributes.get(ATTR_SUPPORTED_FEATURES) == 40
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5199018)